codegen/cpp/fletchgen/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ FetchContent_MakeAvailable(cmake-modules)
 FetchContent_Declare(
   cerata
   GIT_REPOSITORY https://github.com/abs-tudelft/cerata.git
-  GIT_TAG 0.0.11)
```

Comment:
Had to change this tag to `develop` to make CI green, if we want we can create a new tag in cerata and then use that. It needs to be updated because a dependency of cerata used a very old cmake that is now incompatible.